src/main/java/ua/shpp/eqbot/utility/ConverterDTO.java

```diff
@@ -0,0 +1,24 @@
+package ua.shpp.eqbot.utility;
+
+import org.modelmapper.ModelMapper;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import ua.shpp.eqbot.dto.UserDto;
+import ua.shpp.eqbot.model.UserEntity;
+
+public class ConverterDTO {
+    private static final ModelMapper modeMODEL_MAPPERMapper = new ModelMapper();
```

Comment:
We really need a name like modeMODEL_MAPPERMapper

Reply:
omg! sorry